### PR TITLE
docs: add santosh yadav as GDE resource

### PR DESCRIPTION
**Pull request by santoshyadavdev**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by gkalpak — APPROVED** [review]

File: aio/content/marketing/contributors.json [hunk 1]
```diff
@@ -856,5 +856,13 @@
     "bio": "Doug is an overly-opinionated software developer with a passion for making awesome developer tools. He is motivated primarily by his selfish desire to make his own life easier, but loves to share his tools and workflows with others to make everyone's lives easier.",
     "groups": ["Angular"],
     "lead": "igorminar"
+  },
+  "santosh": {
+    "name": "Santosh Yadav",
+    "picture": "santoshyadav.jpg",
+    "twitter": "Santosh19742211",
+    "website": "https://www.santoshyadav.dev",
+    "bio": "Santosh is GDE for Angular and Web Technologies and loves to contribute to Open Source. He is the creator of ng deploy for netlify and core team member for NestJS Addons. He writes for AngularInDepth and mentors for DotNetTricks, he is the organizer for Pune Tech Meetup and conducts the free workshops on Angular.",
```
Comment (kapunahelewong) [inline aio/content/marketing/contributors.json]:
```suggestion
    "bio": "Santosh is a GDE for Angular and Web Technologies and loves to contribute to Open Source. He is the creator of ng deploy for netlify and core team member for NestJS Addons. He writes for AngularInDepth, mentors for DotNetTricks, organizes the Pune Tech Meetup, and conducts free workshops on Angular.",
```

For "organizes the Pune Tech Meetup", "the" is optional if its official name is "Pune Tech Meetup", but if it's called "Tech Meetup" that happens to be located in Pune, then you'd definitely want "the". Having "the" can help with clarity in either case.

**Review by kapunahelewong — COMMENTED** [review]
A few small suggestions. Congrats, @santoshyadav198613!

File: aio/content/marketing/contributors.json (hunk 1 above)
Comment (santoshyadav198613) [reply aio/content/marketing/contributors.json]: Thanks @kapunahelewong and done with changes.

**Review by StephenFluin — APPROVED** [review]

**Comment by kapunahelewong** [comment]
Hey @santoshyadav198613, could you try [rebasing](https://gist.github.com/kapunahelewong/567c30f7b353fd1c39bbf6d8343e2dca#how-to-rebase)? I think that will fix the failures. Please tag me when you're ready. I'd love to mark for merge. 🚀

**Comment by santoshyadav198613** [comment]
Done @kapunahelewong .

**Review by kapunahelewong — APPROVED** [review]

**Pull request closed without merging**